src/SpinWaveTheory/SWTCalculations.jl

```diff
@@ -415,8 +469,6 @@ function dssf(swt::SpinWaveTheory, qs)
     Nf = sys.mode == :SUN ? Ns-1 : 1
     N  = Nf + 1
     nmodes  = Nf * Nm 
-    M = sys.mode == :SUN ? 1 : (Ns-1) # scaling factor (=1) if in the fundamental representation
-    sqrt_M = √M
```

Comment:
Side note: this change will need to be propagated to the relevant `intensity_formula` function (which cannibalizes `dssf`) during the merge

Reply:
Hi Sam. Yes, this is true. We should keep this in mind.